libs/imap/patches/1002_fix_ftbfs.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/osdep/unix/flocklnx.c
++++ b/src/osdep/unix/flocklnx.c
+@@ -57,7 +57,7 @@
+     case ENOLCK:		/* lock table is full */
+       sprintf (tmp,"File locking failure: %s",strerror (errno));
+       mm_log (tmp,WARN);	/* give the user a warning of what happened */
+-      if (!logged++) syslog (LOG_ERR,tmp);
++	  if (!logged++) syslog ("%s",LOG_ERR,tmp);
```

Comment:
Wrong order... `syslog(LOG_ERR, "%s", tmp);`

Reply:
I'll see into it, just added the debian default patches, no change to them

Reply:
Well, it's wrong.  See [syslog](https://linux.die.net/man/3/syslog).